apps/e2e/src/engine-client/signerAndOrderTests.ts

```diff
@@ -1,130 +1,44 @@
 import {
   createDeterministicLinkedSignerPrivateKey,
-  depositCollateral,
   getIsolatedOrderDigest,
   getOrderDigest,
   getOrderNonce,
-  subaccountFromBytes32,
-  subaccountFromHex,
-  subaccountToBytes32,
   subaccountToHex,
   VERTEX_ABIS,
-  VLP_PRODUCT_ID,
 } from '@vertex-protocol/contracts';
-import { MOCK_ERC20_ABI } from '@vertex-protocol/contracts/dist/common/abis/MockERC20';
 import {
   EngineClient,
-  EngineIsolatedOrderParams,
   EngineOrderParams,
+  EngineIsolatedOrderParams,
 } from '@vertex-protocol/engine-client';
-import {
-  BigDecimals,
-  addDecimals,
-  removeDecimals,
-  toBigDecimal,
-  toBigInt,
-} from '@vertex-protocol/utils';
+import { addDecimals, toBigDecimal } from '@vertex-protocol/utils';
 import { createWalletClient, getContract, http, zeroAddress } from 'viem';
 import { privateKeyToAccount } from 'viem/accounts';
 import { getExpiration } from '../utils/getExpiration';
 import { prettyPrint } from '../utils/prettyPrint';
-import { runWithContext } from '../utils/runWithContext';
 import { RunContext } from '../utils/types';
-import { waitForTransaction } from '../utils/waitForTransaction';
+import { runWithContext } from '../utils/runWithContext';
 
-async function fullSanity(context: RunContext) {
+export async function signerAndOrderTests(context: RunContext) {
```

Comment:
Should tests be assuming that a subaccount exists with collateral? Otherwise, we should be depositing collateral at the beginning of the tests and withdrawing a roughly equivalent amt at the end.

This might introduce a lot of boilerplate though, so we could also have some `setup` script that mints & deposits to create an account